lib/active_fedora/rdf/field_map_entry.rb

```diff
@@ -15,14 +16,15 @@ def initialize
     # Merges any existing values for solr fields with new, incoming values and ensures that resulting values are unique.
     # @param [Symbol] type the data type for the field such as :string, :date, :integer
     # @param [Array] behaviors Solrizer's behaviors for indexing such as :stored_searhable, :symbol
-    # @param [Array] values existing values for the solr field
-    def merge!(type, behaviors, values)
+    # @param [Array] new_values existing values for the solr field
```

Comment:
@awead pinging you as you were the original author ( https://github.com/projecthydra/active_fedora/commit/4d46e30dbdde378e1d480265b0f985a34fd0be30).  I think the word "existing" is confusing and could be eliminated. How about "values to add to the solr field"?